tests/e2e/test_flask.py

```
@@ -0,0 +1,100 @@
+    """E2Eテスト用にFlaskサーバーをサブプロセスで起動する
+
+    Yields:
+        None
+    """
+    # 現在の環境変数をコピーして、テスト用の変数を追加
+    env = os.environ.copy()
+    env.update({
+        "CONTAINER_PORT": "5000",
+        "PYTEST_CURRENT_TEST": "1",
+    })
+    
```

Ruff (0.11.9)

Blank line contains whitespace

(W293)